<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BootstrapFibersToDistancesForm</class>
 <widget class="QWidget" name="BootstrapFibersToDistancesForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelFibers">
     <property name="text">
      <string>Fibers</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboFibers"/>
   </item>
   <item>
    <widget class="QLabel" name="labelFiberIds">
     <property name="text">
      <string>Fiber ID's</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboFiberIds"/>
   </item>
   <item>
    <widget class="QLabel" name="labelDistanceMeasure">
     <property name="text">
      <string>Distance Measure</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboDistanceMeasure"/>
   </item>
   <item>
    <widget class="QLabel" name="labelDistanceTable">
     <property name="text">
      <string>Distance Table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboDistanceTable"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonLoad">
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="buttonCompute">
     <property name="text">
      <string>Compute Distances</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>222</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
